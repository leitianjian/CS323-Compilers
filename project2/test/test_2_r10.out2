Error type 10 at Line 17: applying indexing operator ([...]) on non-array type variables
Error type 8 at Line 17: function return type is not matched
