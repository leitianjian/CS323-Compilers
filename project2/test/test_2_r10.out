Error type 10 at line 17: applying indexing operator ([...]) on non-array type variables
